src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdClientSettings.Edit.cshtml

```diff
@@ -13,209 +14,210 @@
+        <div class="form-check">
+            <input type="checkbox" class="form-check-input" asp-for="UseIdTokenFlow">
+            <label class="form-check-label" asp-for="UseIdTokenFlow">@T["Use 'id_token' response type"]</label>
+            <span class="hint dashed">@T["Click <a href=\"{0}\" target=\"_blank\">here</a> for an example.", "http://openid.net/specs/openid-connect-core-1_0.html#id_tokenExample"]</span>
+        </div>
     </div>
-</div>
 
-<div class="mb-3" asp-validation-class-for="UseIdTokenTokenFlow">
-    <div class="form-check">
-        <input type="checkbox" class="form-check-input" asp-for="UseIdTokenTokenFlow">
-        <label class="form-check-label" asp-for="UseIdTokenTokenFlow">@T["Use 'id_token token' response type"]</label>
-        <span class="hint dashed">
-            <a href="http://openid.net/specs/openid-connect-core-1_0.html#id_token-tokenExample" target="_blank">@T["example:"] http://openid.net/specs/openid-connect-core-1_0.html#id_token-tokenExample</a>
-        </span>
+    <div class="mb-3" asp-validation-class-for="UseIdTokenTokenFlow">
+        <div class="form-check">
+            <input type="checkbox" class="form-check-input" asp-for="UseIdTokenTokenFlow">
+            <label class="form-check-label" asp-for="UseIdTokenTokenFlow">@T["Use 'id_token token' response type"]</label>
+            <span class="hint dashed">@T["Click <a href=\"{0}\" target=\"_blank\">here</a> for an example.", "http://openid.net/specs/openid-connect-core-1_0.html#id_token-tokenExample"]</span>
+        </div>
     </div>
-</div>
 
-<div class="mb-3" asp-validation-class-for="ResponseMode">
-    <label asp-for="ResponseMode" class="form-label">@T["Response Mode"]</label>
-    <select asp-for="ResponseMode" class="form-select">
-        <option id="@OpenIdConnectResponseMode.FormPost" value="@OpenIdConnectResponseMode.FormPost">@T[OpenIdConnectResponseMode.FormPost]</option>
-        @*<option id="@OpenIdConnectResponseMode.Fragment" value="@OpenIdConnectResponseMode.Fragment">@T[OpenIdConnectResponseMode.Fragment]</option>*@
-        <option id="@OpenIdConnectResponseMode.Query" value="@OpenIdConnectResponseMode.Query">@T[OpenIdConnectResponseMode.Query]</option>
-    </select>
-    <span asp-validation-for="ResponseMode"></span>
-    <span class="hint">@T["Configure Response Mode see: http://openid.net/specs/openid-connect-core-1_0.html#ImplicitAuthResponse"]</span>
-</div>
+    <div class="mb-3" asp-validation-class-for="ResponseMode">
+        <label asp-for="ResponseMode" class="form-label">@T["Response Mode"]</label>
+        <select asp-for="ResponseMode" class="form-select">
+            <option id="@OpenIdConnectResponseMode.FormPost" value="@OpenIdConnectResponseMode.FormPost">@T[OpenIdConnectResponseMode.FormPost]</option>
+            @*<option id="@OpenIdConnectResponseMode.Fragment" value="@OpenIdConnectResponseMode.Fragment">@T[OpenIdConnectResponseMode.Fragment]</option>*@
```

Comment:
Let's remove it, it's not something we'll ever support.

Reply:
done.